pandas/tests/groupby/test_apply.py

```diff
@@ -868,13 +868,15 @@ def test_apply_multi_level_name(category):
     b = [1, 2] * 5
     if category:
         b = pd.Categorical(b, categories=[1, 2, 3])
+        expected_index = pd.CategoricalIndex([1, 2], categories=[1, 2, 3], name="B")
```

Comment:
Is there an open issue for this?

Reply:
I don't believe so. There is only one issue tagged with categorical, groupby, and apply which is not relevant. I also took a  look through those tagged as categorical and groupby and didn't see anything either.

Reply:
Can you add a whatsnew for this? Something along the lines of `groupby apply will now maintain a CategoricalIndex` (assuming that is now the case)